plans/dht/test/set-up.go

```diff
@@ -16,6 +17,38 @@ import (
 
 // SetUp sets up the elements necessary for the test cases
 func SetUp(ctx context.Context, runenv *runtime.RunEnv, timeout time.Duration, randomWalk bool, bucketSize int, autoRefresh bool, watcher *sync.Watcher, writer *sync.Writer) (host.Host, *kaddht.IpfsDHT, []peer.AddrInfo, error) {
+	// TODO: just put the hostname inside the runenv?
+	hostname, err := os.Hostname()
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	if runenv.TestSidecar {
```

Comment:
This way, we can just not run a sidecar when running locally. We can still add support, we'd just have to re-implement parts of docker (manage network namespaces).